signals.mss

```diff
@@ -669,8 +671,12 @@ Format details:
       text-face-name: @bold-fonts;
       text-size: 10;
     }
-  }
 
+    /* triangle-shaped stop marker */
+    ["feature"="NL:227b"]["shape"="triangle"] {
```

Comment:
train_protection_shape

Reply:
Your selector `["feature"="NL:227b"]["shape"="triangle"]` will not match anything because line 659 contains `["shape"="arrow"]`. Remove `["shape"="arrow"]` to make it work.